amplify-tools-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'groovy'
```

Comment:
Please add the copyright header that other `.gradle` files in this repo are using